src/Microsoft.HttpRepl.Tests/Commands/DeleteCommandsTest.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Tests.SampleApi;
+using Microsoft.Repl;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class DeleteCommandTests : IClassFixture<DeleteCommandsFixture>
+    {
+        private readonly DeleteCommandsConfig _config;
+        public DeleteCommandTests(DeleteCommandsFixture DeleteCommandsFixture)
+        {
+            _config = DeleteCommandsFixture.Config;
+        }
+
+        [Fact]
+        public async Task ExecuteAsync_WithNoBasePath_VerifyError()
+        {
+            HttpState httpState = new HttpState();
+
+            string expectedErrorMessage = Resources.Strings.Error_NoBasePath.SetColor(httpState.ErrorColor);
+            string actualErrorMessage = null;
+
+            IShellState shellState = MockHelpers.GetMockedShellState(errorMessageCallback: (s) => actualErrorMessage = s);
+
+            DeleteCommand command = new DeleteCommand();
+            ICoreParseResult parseResult = CoreParseResultHelper.Create("DELETE");
+            CancellationTokenSource cts = new CancellationTokenSource();
```

Comment:
nit: since you're not cancelling this token anywhere, could could use `CancellationToken.None` instead of declaring the CTS.

Reply:
Done.